src/index.js

```diff
@@ -15,28 +16,42 @@ class App extends React.Component {
     this.removeFromOrder = this.removeFromOrder.bind(this);
   }
 
-  addToOrder(name, price, bread, isHot) {
-    this.setState(prevState => ({
-      orderItems: [...prevState.orderItems, { name, price, bread, isHot }],
-      orderTotal: prevState.orderTotal + price
-    }));
-  }
-
-  removeFromOrder(i) {
-    this.setState(prevState => ({
-      orderItems: prevState.orderItems.filter((_, idx) => idx !== i)
-    }));
-    this.setState(prevState => {
+  componentDidMount() {
+    const orderRef = firebase.database().ref('order');
+    orderRef.on('value', (snapshot) => {
+      let items = snapshot.val();
+      let newState = [];
```

Comment:
This could be a `const` too man and I'd avoid calling it anything to do with state, name it what it is rather than where its going